pom.xml

```diff
@@ -72,7 +72,7 @@
     <avro.version>1.8.2</avro.version>
     <bigquery.connector.hadoop2.version>hadoop2-1.0.0</bigquery.connector.hadoop2.version>
     <commons.codec.version>1.4</commons.codec.version>
-    <cdap.version>6.7.0</cdap.version>
+    <cdap.version>6.8.0-SNAPSHOT</cdap.version>
```

Comment:
why are we bumping up cdap version depedency? Are we adding a dependency on a new CDAP API?

Reply:
@snehalakshmisha made some changes in `cdap-api`, and `hydrator-common`. After those changes were merged with develop. My pipelines started breaking and I had to update cdap version dependency for it to work again.